docs/management/commands/update_docs.py

```diff
@@ -213,22 +214,23 @@ def build_doc_release(self, release, force=False, interactive=False):
             conf_extensions = Config.read(source_dir.resolve()).extensions
             extensions = [*conf_extensions, "docs.builder"]
             try:
-                Sphinx(
-                    srcdir=source_dir,
-                    confdir=source_dir,
-                    outdir=build_dir,
-                    doctreedir=build_dir.joinpath(".doctrees"),
-                    buildername=builder,
-                    # Translated docs builds generate a lot of warnings, so send
-                    # stderr to stdout to be logged (rather than generating an email)
-                    warning=sys.stdout,
-                    parallel=multiprocessing.cpu_count(),
-                    verbosity=self.verbosity,
-                    confoverrides={
-                        "language": to_locale(release.lang),
-                        "extensions": extensions,
-                    },
-                ).build()
+                with patch_docutils(source_dir), docutils_namespace():
```

Comment:
I'd add a comment here, for the sake of future maintainer (aka you or me in 3 months 😁 ):
```suggestion
                # Prevent global state persisting between builds
                # https://github.com/sphinx-doc/sphinx/issues/12130
                with patch_docutils(source_dir), docutils_namespace():
```